src/Storages/System/attachSystemTables.cpp

```diff
@@ -96,105 +96,105 @@ namespace DB
-    attach<SystemMergeTreeSettings<false>>(context, system_database, "merge_tree_settings");
-    attach<SystemMergeTreeSettings<true>>(context, system_database, "replicated_merge_tree_settings");
-    attach<StorageSystemBuildOptions>(context, system_database, "build_options");
-    attach<StorageSystemFormats>(context, system_database, "formats");
-    attach<StorageSystemTableFunctions>(context, system_database, "table_functions");
-    attach<StorageSystemAggregateFunctionCombinators>(context, system_database, "aggregate_function_combinators");
-    attach<StorageSystemDataTypeFamilies>(context, system_database, "data_type_families");
-    attach<StorageSystemCollations>(context, system_database, "collations");
-    attach<StorageSystemTableEngines>(context, system_database, "table_engines");
-    attach<StorageSystemContributors>(context, system_database, "contributors");
-    attach<StorageSystemUsers>(context, system_database, "users");
-    attach<StorageSystemRoles>(context, system_database, "roles");
-    attach<StorageSystemGrants>(context, system_database, "grants");
-    attach<StorageSystemRoleGrants>(context, system_database, "role_grants");
-    attach<StorageSystemCurrentRoles>(context, system_database, "current_roles");
-    attach<StorageSystemEnabledRoles>(context, system_database, "enabled_roles");
-    attach<StorageSystemSettingsProfiles>(context, system_database, "settings_profiles");
-    attach<StorageSystemSettingsProfileElements>(context, system_database, "settings_profile_elements");
-    attach<StorageSystemRowPolicies>(context, system_database, "row_policies");
-    attach<StorageSystemQuotas>(context, system_database, "quotas");
-    attach<StorageSystemQuotaLimits>(context, system_database, "quota_limits");
-    attach<StorageSystemQuotaUsage>(context, system_database, "quota_usage");
-    attach<StorageSystemQuotasUsage>(context, system_database, "quotas_usage");
-    attach<StorageSystemUserDirectories>(context, system_database, "user_directories");
-    attach<StorageSystemPrivileges>(context, system_database, "privileges");
-    attach<StorageSystemErrors>(context, system_database, "errors");
-    attach<StorageSystemWarnings>(context, system_database, "warnings");
-    attach<StorageSystemDataSkippingIndices>(context, system_database, "data_skipping_indices");
-    attach<StorageSystemLicenses>(context, system_database, "licenses");
-    attach<StorageSystemTimeZones>(context, system_database, "time_zones");
-    attach<StorageSystemBackups>(context, system_database, "backups");
-    attach<StorageSystemSchemaInferenceCache>(context, system_database, "schema_inference_cache");
-    attach<StorageSystemDroppedTables>(context, system_database, "dropped_tables");
+    attach<StorageSystemOne>(context, system_database, "one", "Used when the table is not specified explicitly, for example in queries like `SELECT 1`. Analog of the DUAL table in Oracle and MySQL.");
```

Comment:
Comment does not say what the system.one actually does. Could include "This table contains a single row with a single dummy UInt8 column containing the value 0."

Reply:
(I'd also omit the reference to DUAL tables. This is interesting but belongs IMHO more into the official docs.)